Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell.swift

```diff
@@ -39,11 +39,10 @@ open class _ChatMessageComposerCommandCellView<ExtraData: ExtraDataTypes>: View,
         backgroundColor = .clear
```

Comment:
`backgroundColor = uiConfig.colorPalette.popoverBackground` Please set the background color to this, to be synched with `ChatMessageComposerMentionCollectionViewCell`

Reply:
Is this really needed? The background color will be the one from the suggestions container. Why should we force the cell to have a bg color?

Reply:
Hmm it's good question, I mean I wrote mainly because of consistency and for the snapshots to look ok while taking a look at them. Maybe it's not that important.